swap/swap_test.go

```diff
@@ -744,6 +749,14 @@ func TestDebtCheques(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	// ...on which we wait until the cashCheque is actually terminated (ensures proper nounce count)
```

Comment:
nounce -> nonce?

Reply:
I copied that comment from elsewhere where it already had this mistake. I fixed it in all places.